app.js

```diff
@@ -338,6 +338,7 @@ function fadeIn(element, duration) {
   };
   tick();
 }
+let isGameStarted = false
```

Comment:
Please move this variablle to the top of the file, where we decllare other variables like `let gameFps`.